src/views/tickets/orders.php

```diff
@@ -44,7 +44,7 @@
 	<form method="post">
 
 	<?php tribe_tickets_get_template_part( 'tickets/orders-rsvp' ); ?>
-	<?php tribe_tickets_get_template_part( 'tickets/orders-tickets' ); ?>
+	<?php tribe_tickets_get_template_part( 'tickets-plus/orders-tickets' ); ?>
```

Comment:
This definitely fixes things but I don't really feel we should have this call in here at all. 

It's fine for Event Tickets Plus (or any dependent plugin) to call code in Event Tickets (or other base/parent plugin), because that's the way the dependencies flow, but it's not ideal to do that in reverse and have Event Tickets calling Event Tickets Plus.

I realize you didn't introduce this pattern in this PR, but could we use an action/callback approach instead?